asv_bench/benchmarks/tslibs/strftime.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+
+import pandas as pd
```

Comment:
sorry for the late review; these benchmarks don't belong in the tslibs directory (unless you can re-write them to not depend on anything outside of tslibs).  can you do a follow-up to move them?

Reply:
that would do it, yes.  thanks

Reply:
Done in #47665 . Let's wait for a successful build just in case..